scripts/mkcloud

```diff
@@ -1291,6 +1305,12 @@ function sanity_checks
         fi
     fi
 
+    if [[ $wantedcmds =~ "setupironicnodes" ]]; then
```

Comment:
I'm not sure about that... `nodenumberironicnode` is needed only if we're using `setupironicnodes`. You don't need to use `setupironicnodes` at all (e.g. if you have some real baremetal testing machines or some manually created test VMs).